src/script/storage/StorageService.ts

```diff
@@ -68,12 +79,6 @@ export class StorageService {
 
     this.db = new Dexie(this.dbName);
 
-    this.db.on('blocked', event => {
-      if (event.dataLoss !== 'none') {
-        this.logger.error('Database is blocked', event);
-      }
-    });
-
     this._upgradeStores(this.db);
 
     return this.engine
```

Comment:
Why no `async` here?

Reply:
The JS code was not using `async/await`, so I didn't want to apply it in order to make it easier to compare both versions during the PR review.

Reply:
I see. But in `StorageRepository.ts` you also introduced `async`. I think it's still readable if you convert this one to async :+1:

Reply:
Ok, Conversion done with: https://github.com/wireapp/wire-webapp/pull/6887/commits/5a8b6d9d8d61fb3775ae588c112648123c5c96df